src/api.c

```diff
@@ -61,6 +61,7 @@ struct nmd_ctx {
     int64_t last_frame_poped_ts;
     int64_t first_ts;
     int64_t last_ts;
+    int loop;
```

Comment:
I think I actually prefer `eof` with the following comment: "set if the latest NULL frame returned meant EOF"

Reply:
Done.